scopes/ui-foundation/ui/ui.main.runtime.ts

```diff
@@ -258,25 +258,25 @@ export class UiMain {
   /**
    * create a Bit UI runtime.
    */
-  async createRuntime({ uiRootName, pattern, dev, port, rebuild, verbose, skipUiBuild }: RuntimeOptions) {
-    const maybeUiRoot = this.getUi(uiRootName);
-    if (!maybeUiRoot) throw new UnknownUI(uiRootName, this.possibleUis());
+  async createRuntime({ uiRootAspectIdOrName, pattern, dev, port, rebuild, verbose, skipUiBuild }: RuntimeOptions) {
```

Comment:
This is a breaking change, it might be used by the cloud with the old name, so I think it's better to also support the old name (you can make it deprecated, and just set the `uiRootAspectIdOrName` if only the old name was passed.
I'm still checking if it's in use in the cloud. but better to be safe for public APIs.